### ISSUE TITLE
Minification breaks the calc function if using a + operator

### ISSUE THREAD

**Issue by empyrical** [ndparker/rcssmin#1]
For example, if you do this:

``` css
body {
    width: calc(90% + 10px);
}
```

rcssmin will output:

``` css
body{width:calc(90%+10px)}
```

This breaks things, because  the browser wants the spaces to stay around +. A temporary work-around, if people want it, that doesn't get the required spaces removed is a double negative:

``` css
body {
    width: calc(90% - -10px);
}
```

which outputs:

``` css
body{width:calc(90% - -10px)}
```

**Comment by ndparker** [comment]
Thanks! I'll look into it.

**Comment by ndparker** [comment]
N.B.: Do you have any information, which browser we're talking about? I've just read the specification, which explicitly allows using calc() without any spacing.

**Comment by ndparker** [comment]
Ok, scratch that. Missed the relevant paragraph ;-)

**Comment by GiovanniFrigo** [comment]
This issue is still here. Any idea on when it will be fixed?

**Comment by ndparker** [comment]
I'm actually working on it (in little timeslices, though). It's harder than it looks, because calc can be nested indefinitely, which doesn't work very well with regexes.
I'm still thinking about how to simplify the problem at best.

Ideas welcome :-)

**Comment by ndparker** [comment]
preliminary fix applied in 1.0.6 (just released)

**Comment by GiovanniFrigo** [comment]
Great news, thanks!
